rbac/ledger_sync/database.py

```diff
@@ -136,3 +137,37 @@ def run_query(self, query):
         """Takes a query based on get_table, and runs it.
         """
         return query.run(self._conn)
+
+    def clear_role(self, role_id, update_time):
+        """Takes a role ID and update_time and removes all role admins,
+        role owners, role members, and base role objects in rethinkDB that are
+        older than the update_time.
+        """
+        update_time = r.epoch_time(int(update_time) - 1)
+        # remove all old entries in role_attributes with role_id
+        (
+            r.db(self._name)
+            .table("role_members")
+            .filter({"role_id": role_id})
+            .filter(lambda role_member: role_member["created_date"] < update_time)
```

Comment:
Is the bug where created_date is actually the last updated time documented anywhere? If not, let's leave a note here so when another developer reads this code, they will know why you are comparing created_date with the updated time.

Reply:
fair, I'll leave a `NOTE:` tag with an explanation and rebase